<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerSubtraction</class>
 <widget class="QWidget" name="ContainerSubtraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>970</width>
        <height>745</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="fileInput">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gbTest">
          <item row="0" column="0">
           <widget class="QLabel" name="sampleLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sample</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="containerLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Container</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoLoad" stdset="0">
             <bool>true</bool>
            </property>
            <property name="workspaceSuffixes" stdset="0">
             <stringlist>
              <string>_red</string>
              <string>_sqw</string>
              <string>_elf</string>
             </stringlist>
            </property>
            <property name="fileBrowserSuffixes" stdset="0">
             <stringlist>
              <string>_red.nxs</string>
              <string>_sqw.nxs</string>
              <string>_elf.nxs</string>
             </stringlist>
            </property>
            <property name="showLoad" stdset="0">
             <bool>false</bool>
            </property>
            <property name="ShowGroups" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="MantidQt::MantidWidgets::DataSelector" name="dsContainer" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoLoad" stdset="0">
             <bool>true</bool>
            </property>
            <property name="workspaceSuffixes" stdset="0">
             <stringlist>
              <string>_red</string>
              <string>_sqw</string>
              <string>_elf</string>
             </stringlist>
            </property>
            <property name="fileBrowserSuffixes" stdset="0">
             <stringlist>
              <string>_red.nxs</string>
              <string>_sqw.nxs</string>
              <string>_elf.nxs</string>
             </stringlist>
            </property>
            <property name="showLoad" stdset="0">
             <bool>false</bool>
            </property>
            <property name="ShowGroups" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbOptions">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="ckScaleCan">
            <property name="text">
             <string>Scale Container by factor:</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="loScaleFactor">
            <item>
             <widget class="QDoubleSpinBox" name="spCanScale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="ckShiftCan">
            <property name="text">
             <string>Shift x-values of container by adding:</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="loShift">
            <item>
             <widget class="QDoubleSpinBox" name="spShift">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="minimum">
               <double>-999.990000000000009</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="title">
          <string>Preview</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
          <item>
           <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPreview" native="true">
            <property name="showLegend" stdset="0">
             <bool>true</bool>
            </property>
            <property name="canvasColour" stdset="0">
             <color>
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lbPreviewSpec">
              <property name="text">
               <string>Spectrum: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spPreviewSpec"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbPlotPreview">
              <property name="text">
               <string>Plot Current Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MantidQt::CustomInterfaces::RunView" name="runWidget" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="gbOutput">
         <property name="title">
          <string>Output Options</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="MantidQt::CustomInterfaces::OutputPlotOptionsView" name="ipoPlotOptions" native="true"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save Result</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::CustomInterfaces::OutputPlotOptionsView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Spectroscopy/OutputWidget/OutputPlotOptionsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::RunView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Spectroscopy/RunWidget/RunView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ckScaleCan</sender>
   <signal>toggled(bool)</signal>
   <receiver>spCanScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckShiftCan</sender>
   <signal>toggled(bool)</signal>
   <receiver>spShift</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
